src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/RazorTranslateDiagnosticsService.cs

```diff
@@ -36,12 +36,11 @@ internal class RazorTranslateDiagnosticsService(IDocumentMappingService document
     private readonly ILogger _logger = loggerFactory.GetOrCreateLogger<RazorTranslateDiagnosticsService>();
     private readonly IDocumentMappingService _documentMappingService = documentMappingService;
 
-    // Internal for testing
-    internal static readonly IReadOnlyCollection<string> CSharpDiagnosticsToIgnore = new HashSet<string>()
-    {
+    private static readonly HashSet<string> s_cSharpDiagnosticsToIgnore =
+    [
         "RemoveUnnecessaryImportsFixable",
         "IDE0005_gen", // Using directive is unnecessary
-    };
+    ];
```

Comment:
If this is never mutated, consider making it a `FrozenSet<string>`.

Reply:
Good call. It is indeed never mutated, and in fact I considered just making it two if statements, but frozen set works too :)